apiary.apib

```diff
@@ -463,6 +466,7 @@ Partial update of an organization
     + name:POC - Organization name
     + maintainerId - Assign the organization to another maintainer.
     + allowAutoJoin - Set whether superAdmins need approval to join the organization's projects (default `true`).
+    + crmId - Set CRM ID. Only maintainer members and superadmins can change this.
```

Comment:
Když tam nedáš hodnotu tak je pak v apiary Request:
![snimek z 2019-02-12 12-13-12](https://user-images.githubusercontent.com/5272798/52631687-bca3e680-2ebf-11e9-999b-cc0b555d5d8c.png)
A response
![snimek z 2019-02-12 12-13-20](https://user-images.githubusercontent.com/5272798/52631697-c3caf480-2ebf-11e9-9cfa-1dd472f730c9.png)

Vypadá to, že nejde nastavit prázdné crmId. Asi bych tam dal hodnotu nebo upravil response. Nebo je to takhle ok ?

Reply:
upravil jsem